manifests/e/Emurasoft/EmEditor/21.1.1/Emurasoft.EmEditor.installer.yaml

```diff
@@ -0,0 +1,30 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Emurasoft.EmEditor
+PackageVersion: 21.1.1
+MinimumOSVersion: 10.0.0.0
+Platform: 
+- Windows.Desktop
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://emeditor.blob.core.windows.net/emed64_21.1.1.msi
+  InstallerSha256: DA5D5A6B0664E959199265425545713819A9F505A3EE859F48451B59EB9A65CB
+  ProductCode: '{AE83F859-A3B8-41EA-82EF-6F21F692C0B5}'

```

Comment:
```suggestion
  ProductCode: '{F7B23400-A231-43DE-AD63-0B7D87341954}'
```

Reply:
![image](https://user-images.githubusercontent.com/11600822/132052372-874efc1a-15b1-41a9-a18c-ef3dda6a463e.png)

Reply:
'{AE83F859-A3B8-41EA-82EF-6F21F692C0B5}' is correct.

'{F7B23400-A231-43DE-AD63-0B7D87341954}' is the product code for the previous version (21.1.0).

Reply:
Ah, oops, checkout'd the wrong PR.

Reply:
I see the issue now, it shows that you added two versions to the PR which messes the Git history for the tools that checks PRs, probably because you're doing your changes in the master branch.

Reply:
Sorry, I will remove the commit for 21.1.0.

Reply:
Okay, I removed the 21.1.0 commit.